src/System.Device.Gpio.Tests/README.md

```diff
@@ -0,0 +1,39 @@
+# How to run tests on the Raspberry Pi
+
+Prerequisites: 
+- Installed .NET SDK with the "dotnet" executable in the path
+- Locally cloned repository
+
+Build System.Device.Gpio.dll:
+```
+cd src/System.Device.Gpio
```

Comment:
Instead of this, could you simply tell people to build the whole repo by `build.cmd`/`build.sh` from the root? That will install all SDK dependencies needed in order to be able to run the tests.

Reply:
This doesn't work for me, at least not on the Pi itself (On Windows, I wouldn't use it either, there I just fire up Visual Studio and run the build).


```
pi@raspberrypi:~/projects/iot $ ./build.sh 
 Wiederherstellung in "444.81 ms" für "/home/pi/.nuget/packages/microsoft.dotnet.arcade.sdk/5.0.0-beta.19501.4/tools/Tools.proj" abgeschlossen.
dotnet-install: .NET Core Runtime version 2.1.11 is already installed.
dotnet-install: Adding to current process PATH: `/home/pi/projects/iot/.dotnet`. Note: This change will be visible only when sourcing script.
dotnet-install: Installation finished successfully.
  Wiederherstellung in "30.04 ms" für "/home/pi/projects/iot/eng/common/internal/Tools.csproj" abgeschlossen.
  Wiederherstellung in "7.14 ms" für "/home/pi/projects/iot/samples/force-sensitive-resistor/force-sensitive-resistor.csproj" abgeschlossen.
  Wiederherstellung in "381.32 ms" für "/home/pi/projects/iot/src/Iot.Device.Bindings/Iot.Device.Bindings.csproj" abgeschlossen.
  Wiederherstellung in "513.39 ms" für "/home/pi/projects/iot/src/System.Device.Gpio/System.Device.Gpio.csproj" abgeschlossen.
  <snip many more package restore lines>
Wiederherstellung in "2.6 ms" für "/home/pi/projects/iot/src/System.Device.Gpio/System.Device.Gpio.csproj" abgeschlossen.
/home/pi/.nuget/packages/microsoft.net.compilers.toolset/3.1.0-beta1-19127-06/tasks/netcoreapp2.1/Microsoft.CSharp.Core.targets(58,5): error : Failed to load �/CD���, error: /home/pi/projects/iot/.dotnet/shared/Microsoft.NETCore.App/2.1.11/libhostpolicy.so: wrong ELF class: ELFCLASS64 [/home/pi/projects/iot/src/System.Device.Gpio/System.Device.Gpio.csproj]
/home/pi/.nuget/packages/microsoft.net.compilers.toolset/3.1.0-beta1-19127-06/tasks/netcoreapp2.1/Microsoft.CSharp.Core.targets(58,5): error : An error occurred while loading required library libhostpolicy.so from [/home/pi/projects/iot/.dotnet/shared/Microsoft.NETCore.App/2.1.11] [/home/pi/projects/iot/src/System.Device.Gpio/System.Device.Gpio.csproj]
/home/pi/.nuget/packages/microsoft.net.compilers.toolset/3.1.0-beta1-19127-06/tasks/netcoreapp2.1/Microsoft.CSharp.Core.targets(58,5): error : Failed to load ��, error: /home/pi/projects/iot/.dotnet/shared/Microsoft.NETCore.App/2.1.11/libhostpolicy.so: wrong ELF class: ELFCLASS64 [/home/pi/projects/iot/src/devices/Units/Units.csproj]
<snip many more similar error messages>
Not sure what's happening here, the Pi4 is running an ordinary Raspbian 32 Bit version. For some reason, the build script tries to link 64 Bit libraries.

Reply:
I think we should provide the publish profiles for the most common build configurations. This simplifies building and deploying directly from with VS. I'd rather put that on a separate PR, though.

Reply:
@pgrawehr can you file an issue on that? (if possible can you temporarily switch language to english to make it easier for us to see the errors?)

Reply:
Yea, will do. I'll try to get a log in english.

Reply:
@krwq : I've created #805 for this. An english log has been added there.